sql/catalyst/src/main/java/org/apache/spark/sql/catalog/v2/Catalogs.java

```diff
@@ -50,8 +51,10 @@ private Catalogs() {
    */
   public static CatalogPlugin load(String name, SQLConf conf)
       throws CatalogNotFoundException, SparkException {
-    String pluginClassName = conf.getConfString("spark.sql.catalog." + name, null);
-    if (pluginClassName == null) {
+    String pluginClassName;
+    try {
+      pluginClassName = conf.getConfString("spark.sql.catalog." + name);
```

Comment:
Should we merge @gengliangwang's PR with this change instead? I thought it may be superseded by @cloud-fan's update to configure the v2 session catalog, but since that is delayed, we might want to go ahead with that PR.

Reply:
I just want that fixed :) We can merge @gengliangwang PR instead